app/src/main/java/rightmesh/left/io/gpstracker/Constants.java

```diff
@@ -5,5 +5,6 @@
     public static final String SUPER_PEER_ID = "0x656284abf20af4192d86f2f6f3e7ce04e5718302";
 
     // TODO: fill in with your SuperPeer URL
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
```

Comment:
Rather than using this, maybe put the IP in the buildconfig?
https://stackoverflow.com/questions/44963447/avoiding-hardcoded-ip-addresses-in-android
Not a big deal, just a thought (I hate ignoring warnings)

Reply:
Great place to keep configs. I moved all configs to that place